packages/client/katello-pull-provider-migrate/katello-pull-provider-migrate.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+# set SYSCONFDIR to /etc if it is not set
+if [ -z "$SYSCONFDIR" ]; then
+    SYSCONFDIR=/etc
+fi
+
+# see if /etc/rhsm/rhsm.conf exists
+RHSMCONF_PATH=$SYSCONFDIR/rhsm/rhsm.conf
+if [ -f $RHSMCONF_PATH ]; then
+    echo "Getting configuration from $RHSMCONF_PATH..."
+    # get cert-file and key-file from rhsm.conf
+    CONSUMER_CERT_DIR=$(cat ${RHSMCONF_PATH} | grep consumerCertDir | cut -f3 -d ' ')
+    CERT_FILE=$CONSUMER_CERT_DIR/cert.pem
+    KEY_FILE=$CONSUMER_CERT_DIR/key.pem
+    # get server name from rhsm.conf
+    SERVER_NAME=$(cat $RHSMCONF_PATH | grep -m1 'hostname = ' | cut -f3 -d ' ')
+else
+    echo "$RHSMCONF_PATH not found; exiting"
+    exit 1
+fi
+
+# fail if no server name, cert file or key file
+if [ -z "$SERVER_NAME" ] || [ -z "$CONSUMER_CERT_DIR" ]; then
+    echo "Unable to determine hostname or consumerCertDir from $SYSCONFDIR/rhsm/rhsm.conf; exiting"
+    exit 1
+fi
+
+# fail if client is not registered to a Satellite
+if [ $(grep -c "prefix = /rhsm" $RHSMCONF_PATH) -eq 0 ]; then
```

Comment:
`prefix=/rhsm` is valid syntax too…

and you probably want
```
if ! grep -q '^prefix\s*=\s*/rhsm'; then
```

Reply:
you forgot to add $RHSMCONF_PATH here and I was going nuts trying to figure out why grep wasn't exiting.. ha

Reply:
Hah, never said it's valid syntax 🙈